src/components/screens/monitor/delegates/forgingDetails.css

```diff
@@ -0,0 +1,37 @@
+@import '../../../../app/variables.css';
+@import '../../../../app/mixins.css';
+
+.contentWrapper {
+  & h3 {
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 20px;
```

Comment:
💅 This kind of stuff should IMO come from https://github.com/LiskHQ/lisk-desktop/blob/development/src/app/mixins.css

Reply:
✅